crepecake/src/integration-test/java/com/google/cloud/tools/crepecake/registry/LocalRegistry.java

```diff
@@ -71,6 +73,8 @@ protected void after() {
 
   /** Runs a command with naive tokenization by whitespace. */
   private void runCommand(String command) throws IOException, InterruptedException {
-    new ProcessBuilder(command.split(" ")).start().waitFor();
+    if (new ProcessBuilder(command.split(" ")).start().waitFor() != 0) {
```

Comment:
Splitting on " " is probably less safe than accepting a List<String> as input. I think runCommand should probably be accepting List<String>.